apps/synapse-portal-framework/src/components/DetailsPage/markdown/MarkdownSynapseFromColumnData.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {
+  MarkdownSynapse,
+  MarkdownCollapseProps,
+  MarkdownCollapse,
+  NoContentAvailable,
+} from 'synapse-react-client'
+import { HeadlineWithLinkDerivedFromEntityId } from '../HeadlineWithLink'
+import { useColumnDataMarkdownProps } from './useColumnDataMarkdownProps'
+
+type MarkdownSynapseFromColumnDataProps = {
+  /**
+   * The column name that contains the table data to render
+   */
+  columnName: string
+  /**
+   * If true, displays a headline labeled with the resolved entity's name
+   */
+  showEntityTitle?: boolean
+  /**
+   * If showEntityTitle is true, also prepend this string before the entity title in the headline
+   */
+  entityTitlePrepend?: string
+  /** If true, show a "No content available" message when no markdown can be found.
+   * @default false
+   */
+  showNoContentAvailable?: boolean
+  /** If true, treat the content as raw markdown
+   * @default false
+   */
+  injectMarkdown?: boolean
```

Comment:
I originally kept `injectMarkdown` but I think that name is an artifact of the previous implementation.
```suggestion
  /** If true, treat the content as raw markdown
   * @default false
   */
  isRawMarkdown?: boolean
```

Reply:
Assume all instances of this name would be replaced